src/plugins/kibana/public/dashboard/services/_saved_dashboard.js

```diff
@@ -56,6 +57,7 @@ module.factory('SavedDashboard', function (courier, config) {
     timeRestore: 'boolean',
     timeTo: 'string',
     timeFrom: 'string',
+    refreshInterval: 'string'
```

Comment:
Ah, didn't know the words were customizable and such. In that case it might make sense to map this field as an object type. I believe Rashid once told me the stringified JSON approach in .kibana pre-dated object fields in ES, or at least his knowledge of them, and they're only strings today for backwards compatibility reasons. If we're adding a new field it makes sense to do it right I think.

Also you'll still want to explicitly pick the fields that get sent to ES, so things like $$hashKey don't get included.